src/Antl/index.js

```diff
@@ -124,6 +124,13 @@ class Antl {
    * @return {Mixed}
    */
   get (key, defaultValue = null) {
+    /**
+     * Make sure key is a string before attempting to split it
+     */
+    if (typeof (key) !== 'string') {
+      return defaultValue
```

Comment:
An empty string returns the default value, so would it be acceptable to return the default value if the key is null or undefined and only throw an exception if the key is not a string/undefined/null? 

This would keep people from having to guard against null or replace it with "" in every case where formatString() is used on a variable containing a key.

Reply:
If don't have a key, then you shouldn't be calling this method at all. Guarding against these sort of use cases is not a good practice, the application code to handle the inconsistencies itself.

Also I am not sure how come you can have the `defaultValue` but don't have the `key`?

Reply:
I was counting formatValue()'s default of null (via get()'s default) as the defaultValue in that case.

The suggested change makes sense from the point of view of having the user do all of the guarding logic in application code. I was missing that context and tried to fill it in from other behavior, specifically what happens with an empty string key.

If the lack of a key is intended to be an error case, it seems like an empty string key should also throw an exception. Is that correct?

Currently, if you provide an empty string key, you will get the defaultValue back due to the use of _.get() with both group and messageKey being null. I added a test to confirm this behavior locally.

I just want to make sure this ends up behaving as intended when I make the changes.

Reply:
Definitively better to throw an exception there. Could you make the change?